openpype/hosts/maya/api/lib.py

```diff
@@ -183,6 +183,47 @@ def maintained_selection():
             cmds.select(clear=True)
 
 
+def get_namespace(node):
+    """Return namespace of given node"""
+    return node.rsplit("|", 1)[-1].rsplit(":", 1)[0]
```

Comment:
I think my original suggestion will fail to correctly report the namespace if the node had no namespace. Here's an improved version:
```suggestion
def get_namespace(node):
    """Return namespace of given node"""
    node_name = node.rsplit("|", 1)[-1]
    if ":" in node_name:
        return node_name.rsplit(":", 1)[0]
    else:
        return ""
```

Example test:
```python
def get_namespace(node):
    """Return namespace of given node"""
    node_name = node.rsplit("|", 1)[-1]
    if ":" in node_name:
        return node_name.rsplit(":", 1)[0]
    else:
        return ""
        
def get_namespace_old(node):
    """Return namespace of given node"""
    return node.rsplit("|", 1)[-1].rsplit(":", 1)[0]
        
print(get_namespace("A"))
# ""
print(get_namespace_old("A"))
# "A"
print(get_namespace("A:B:C"))
# "A:B"
print(get_namespace_old("A:B:C"))
# "A:B"
```